config/settings.py

```diff
@@ -1,38 +1,39 @@
 from os import cpu_count, getenv
 
 """
-To customize any of these values, append a line to config/remote_db_env.py such as:
-os.environ['S3_BUCKET'] = 'bucket_name'
+On processing servers, instead of environment varables, append a line to config/remote_db_env.py,
+such as: os.environ['S3_BUCKET'] = 'bucket_name'
 """
 
-
+# for we retain the old access key in case someone deployed a super old EB server.
 BEIWE_SERVER_AWS_ACCESS_KEY_ID = getenv("BEIWE_SERVER_AWS_ACCESS_KEY_ID") or getenv("S3_ACCESS_CREDENTIALS_USER")
 BEIWE_SERVER_AWS_SECRET_ACCESS_KEY = getenv("BEIWE_SERVER_AWS_SECRET_ACCESS_KEY") or getenv("S3_ACCESS_CREDENTIALS_KEY")
 
 # This is the secret key for the website. Mostly it is used to sign cookies. You should provide a
 #  cryptographically secure string to this value.
 FLASK_SECRET_KEY = getenv("FLASK_SECRET_KEY")
 
-# the name of the s3 bucket that will be used to store user generated data, and backups of local
+# The name of the s3 bucket that will be used to store user generated data, and backups of local
 # database information.
 S3_BUCKET = getenv("S3_BUCKET")
 
 # Domain name for the server
 DOMAIN_NAME = getenv("DOMAIN_NAME")
 
-# A list of email addresses that will receive error emails. This value must be a
-# comma separated list; whitespace before and after addresses will be stripped.
+# A list of email addresses that will receive error emails. This value must be a comma separated
+# list; whitespace before and after addresses will be stripped.
 SYSADMIN_EMAILS = getenv("SYSADMIN_EMAILS")
 
-# Sentry DSNs (optional)
+# Sentry DSNs (technically optional)
 SENTRY_DATA_PROCESSING_DSN = getenv("SENTRY_DATA_PROCESSING_DSN")
 SENTRY_ELASTIC_BEANSTALK_DSN = getenv("SENTRY_ELASTIC_BEANSTALK_DSN")
 SENTRY_JAVASCRIPT_DSN = getenv("SENTRY_JAVASCRIPT_DSN")
 
-# Production/Staging: set to "TRUE" if staging
+# Production/Staging: set to "TRUE" if this is a staging/testing/development server to enable some
+# extra features.
 IS_STAGING = getenv("IS_STAGING") or "PRODUCTION"
-REPORT_DECRYPTION_KEY_ERRORS = getenv("REPORT_DECRYPTION_KEY_ERRORS") or False
-STORE_DECRYPTION_KEY_ERRORS = getenv("STORE_DECRYPTION_KEY_ERRORS") or False
+REPORT_DECRYPTION_KEY_ERRORS = bool(getenv("REPORT_DECRYPTION_KEY_ERRORS", False))
+STORE_DECRYPTION_KEY_ERRORS = bool(getenv("STORE_DECRYPTION_KEY_ERRORS", False))
 
```

Comment:
Fair point, and actually it looks like I've screwed this up on IS_STAGING too.

I'm going to go with this slight modification of the recommendation because I find case-sensitive parameters obnoxious:
`os.getenv('DEBUG', 'false').lower() == 'true'`

I'm sticking IS_STAGING on my todo list, the biggest side effect is that all servers are storing decryption key errors, but I need to review it.  I should probably review parameter read-in general, I've been crappy at it before.

Reply:
I am actually going to entirely remove this staging variable.  There are only 2 uses of IS_STAGING, line error recording and an endpoint to determine if there server is a staging server.  Adding a STORE_DECRYPTION_LINE_ERRORS, removing that endpoint because I needed it exactly once in something like 2015, removing IS_STAGING.  Change in upcoming commit.